.github/workflows/main.yml

```
@@ -22,11 +22,11 @@ jobs:
         run: |
           git config --global user.name "github-actions[bot]"
           git config --global user.email "github-actions[bot]@users.noreply.github.com"
-          if git diff -s --exit-code docs/HLD.pdf; then
+          if git diff -s --exit-code docs/HLD.tex; then
```

Add also check for refs.bib
```suggestion
          if git diff -s --exit-code docs/HLD.tex docs/refs.bib; then
```

---

@SzymonKozl done